src/domain/safe/entities/creation-transaction.entity.ts

```diff
@@ -0,0 +1,11 @@
+import { DataDecoded } from '../../data-decoder/entities/data-decoded.entity';
+
+export interface CreationTransaction {
+  created: Date;
+  creator: string;
+  transactionHash: string;
+  factoryAddress: string | null;
```

Comment:
Is `factoryAddress` nullable? In Swagger it seems it's required (also `masterCopy` and `setupData`, but maybe that's a mistake), and [in the code](https://github.com/safe-global/safe-transaction-service/blob/d1a8aaa9e30902d1028998d109bff5d44be5218c/safe_transaction_service/history/services/safe_service.py#L40) it seems it's not optional 🤔

Reply:
You are right, I take from client-gateway https://github.com/safe-global/safe-client-gateway/blob/main/src/common/models/backend/transactions.rs#L97

Reply:
Done here https://github.com/5afe/safe-client-gateway-nest/pull/243/commits/9e42be09bd60b88af2d1e6a28398bc06f2360b00